server/chat-plugins/lottery.ts

```diff
@@ -209,7 +231,12 @@ export const commands: ChatCommands = {
 			const participants = Object.entries(lottery.participants).map(([ip, participant]) => {
 				return `${participant}${canSeeIps ? ' (IP: ' + ip + ')' : ''}`;
 			});
-			const buf = `<b>List of participants (${participants.length}):</b><p>${participants.join(', ')}</p>`;
+			let buf = '';
+			if (user.can('declare', null, room)) {
+				buf += `<b>List of participants (${participants.length}):</b><p>${participants.join(', ')}</p>`;
+			} else {
+				buf += `${participants.length} participant(s) joined this lottery.`;
```

Comment:
Does just the count even need to be gated on staff? I see this as being like a poll where you can see the number of people who have voted in the poll?

Reply:
Sure, I was thinking of adding the information (max winners and participants numbers) in the page, but I couldn't find a sensible place to add it.